debian/rules

```diff
@@ -57,10 +57,10 @@ override_dh_makeshlibs:
 # Create dbg packages.
 # FIXME: should we just have a rdma-core-dbg package ?
 override_dh_strip:
-	dh_strip -plibibcm1 --dbg-package=libibcm1-dbg
-	dh_strip -plibibumad3 --dbg-package=libibumad3-dbg
-	dh_strip -plibibverbs1 --dbg-package=libibverbs1-dbg
-	dh_strip -plibrdmacm1 --dbg-package=librdmacm1-dbg
+	dh_strip -plibibcm1 --dbgsym-migration='libibcm1-dbg (<< 12-1~)'
+	dh_strip -plibibumad3 --dbgsym-migration='libibumad3-dbg (<< 12-1~)'
+	dh_strip -plibibverbs1 --dbgsym-migration='libibverbs1-dbg (<< 12-1~)'
+	dh_strip -plibrdmacm1 --dbgsym-migration='librdmacm1-dbg (<< 12-1~)'
 
```

Comment:
I recommend to keep the debug packages and stay with debhelper 9 to still support jessie (and xenial). I see no need to rush to use the dbgsym packages and debhelper 10.